starfish/codebook.py

```diff
@@ -425,7 +454,7 @@ def decode_per_hyb_max(self, intensities: IntensityTable) -> IntensityTable:
 
         Notes
         -----
-        If no code matches the per-channel max of a feature, it will be assigned np.nan instead
+        If no code matches the per-channel max of a feature, it will be assigned 'None' instead
```

Comment:
personally, I think this is a bit odd.  `None` is a specific pythonic concept, but this is a string "None".

Reply:
So I really _really_ want to remove this altogether and I'm debating mapping all these values to ints internally. I'm going to wait to do something until I've poked at the xarray devs a bit because the string index handling is something that I don't like and I'm hoping there is a solution I'm not seeing. 

For now, I propose replacing `None`  with a constant in Features (maybe `no_target`?). I'll edit that into #340 when this stack gets merged.

Reply:
Spoke with Deep and I think #347 describes a solution for this.